core-web/libs/sdk/angular/README.md

```diff
@@ -1,229 +1,599 @@
+```
 
-const DOTCMS_CLIENT_CONFIG: ClientConfig = {
-    dotcmsUrl: environment.dotcmsUrl,
-    authToken: environment.authToken,
-    siteId: environment.siteId
-};
+---
+
+## Key Concepts
+
+| Term                  | Description                                                          |
+|----------------------|----------------------------------------------------------------------|
+| `pageAsset`          | The actual page layout data used by the renderer                     |
+| `DotCMSPageComponent`| Type for mapping content types to Angular components                 |
+| UVE                  | Universal Visual Editor, dotCMS's editing interface                  |
+| Editable Page        | A page enhanced via `DotCMSEditablePageService` for real-time updates|
+
+---
+
+## API Reference
+
+### Components
+
+#### DotCMSLayoutBody
+
+**Overview**: Component used to render the layout for a DotCMS page, supporting both production and development modes.
+
+**Inputs**:
+
+| Input        | Type                    | Required | Default        | Description                                     |
+|--------------|-------------------------|----------| ---------------|------------------------------------------------|
+| `page`       | `DotCMSPageAsset`      | ✅        | -              | The page asset containing the layout to render  |
+| `components` | `DotCMSPageComponent`   | ✅        | `{}`           | Map of content type → Angular component         |
+| `mode`       | `DotCMSPageRendererMode`| ❌        | `'production'` | Rendering mode ('production' or 'development')  |
+
+**Usage**:
+
+```typescript
+import { DotCMSPageAsset } from '@dotcms/types';
+import { DotCMSLayoutBody } from '@dotcms/angular/next';
+
+@Component({
+    template: `
+        <dotcms-layout-body
+            [page]="pageAsset()"
+            [components]="components()"
+            mode="development"
+        />
+    `
+})
+export class PageComponent {
+    components = signal<DotCMSPageComponent>({
+        Blog: import('./blog.component').then(c => c.BlogComponent)
+    });
+    pageAsset = signal<DotCMSPageAsset | null>(null);
+
+    constructor(private readonly client: DotCmsClient) {}
+
+    ngOnInit() {
+        this.client.page
+            .get({ url: '/my-page' })
+            .then(({ pageAsset }) => {
+                this.pageAsset.set(pageAsset);
+            });
+    }
+}
 ```
 
-Add this configuration to `ApplicationConfig` in your Angular app.
+**Component Mapping**:
+
+* The key `Blog` is the content type variable in dotCMS
+* The value `BlogComponent` is the Angular component that renders the blog post
+
+**Mode**:
+
+* `development` mode is used for development and debugging
+* `production` mode is used for production
+
+
+**Editor Integration**:
+
+* Automatically detects if it's inside dotCMS UVE
+* Works with `DotCMSEditablePageService` for real-time updates
+
+**Common Issues**:
+
+* Missing component mappings cause unrendered containers
```

Comment:
What happen is we don't pass this? do we log or something?

Reply:
I added a better explanation and a troubleshooting section!